src/scenes/ProgressReports/EditForm/Steps/SubmitReportStep/ConfirmVariantDialog.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Typography,
+} from '@mui/material';
+import { Role } from '~/api/schema/schema';
+import { PromptResponseListFragment } from '~/common/fragments';
+import { useSession } from '~/components/Session';
+import { ProgressReportEditFragment } from '../../ProgressReportEdit.graphql';
+
+interface ConfirmEmptyVariantsProps {
+  report: ProgressReportEditFragment;
+  open: boolean;
+  onContinue: () => void;
+  onCancel: () => void;
+  onUpdateConfirmStatus: () => void;
+}
+
+export const ConfirmEmptyVariants = ({
+  report,
+  open,
+  onContinue,
+  onCancel,
+  onUpdateConfirmStatus,
+}: ConfirmEmptyVariantsProps) => {
+  const { reportFile } = report;
+  const { session } = useSession();
+  const roles = new Set(session?.roles.value);
+  const hasReportProgressFile = !!reportFile.value;
+  const projectMgmtHeader = !hasReportProgressFile && reportFile.canEdit;
+
+  const handleConfirmDialog = () => {
+    onContinue();
+  };
+
+  const handleCloseDialog = () => {
+    onCancel();
+  };
```

Comment:
I don't like handle functions that just forward on to a prop. It's a layer of indirection that just adds complexity without doing anything. Just pass onContinue/Cancel through to the child components.